ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
I don't think we need this file if it is not doing anything. Otherwise, strict type checking and method return hints should be added.

Reply:
Dropped it